docs/_includes/variation-content.html

```diff
@@ -131,32 +131,32 @@ <h3>{{ variation_name }}</h3>
                 <button href="#{{ variation_slug }}-{{ forloop.index }}" class="a-btn a-btn__link u-hidden" data-toggle-details="hide">Hide details</button>
             </div>
 
-            <div class="govuk-tabs u-hidden" data-module="tabs" id="{{ variation_slug }}-{{ forloop.index }}">
-                <ul class="govuk-tabs__list">
+            <div class="m-tabs u-hidden" data-module="tabs" id="{{ variation_slug }}-{{ forloop.index }}">
+                <ul class="m-tabs__list">
```

Comment:
In our flavor of BEM, the syntax is `block_element__modifier`, so this is saying the `__list` modifier on the `m-tabs` block. However, modifiers should appear with their block, so `m-tabs m-tabs__list`. 

A theoretical example is think of a red square block where we modify it to change the background color to blue. `m-square` is the default red square and `m-square m-square__blue` is the modified blue version.

So instead of the `m-tabs__list` modifier, this would be an `m-tabs_list` element. This is an unfortunate mapping. Govuk actually also uses BEM, but for whatever historical reason our version diverged from the official BEM syntax :( Govuk's element designator is `__` and their modifier is `--`, while our element is `_` and modifier is `__`. 

So bottom line is this is an element in both govuk land and ours, but our syntax to specify it as such differs.